x-pack/test/functional/apps/ml/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -82,6 +85,30 @@ export default function ({ getService }: FtrProviderContext) {
             status: 'stopped',
             progress: '100',
           },
+          rowDetails: {
+            jobDetails: [
+              {
+                id: jobId,
```

Comment:
Nit: from what I can see, this `id` is not part of `ExpectedSectionTable` and is never used, so could probably be removed.

Reply:
Updated here [`73c2500` (#98678)](https://github.com/elastic/kibana/pull/98678/commits/73c250092c54c6e76bbc33d5a2a2ac0a9c2ef028)